src/index.js

```diff
@@ -16,9 +16,22 @@ class App extends React.Component {
     constructor(props) {
         super(props);
 
+        var urlAPI, urlDocs;
+        try {
+            // validate url and force trailing slash
+            urlAPI = new URL(process.env.REACT_APP_APIURL).toString();
```

Comment:
This won't work as intended if the registry isn't at the root of a domain (i.e. `example.com/registry`)

Reply:
I added a check for `.endsWith('/')` which should work.